packages/@atjson/offset-annotations/src/annotations/horizontal-rule.ts

```diff
@@ -1,6 +1,6 @@
-import { ObjectAnnotation } from "@atjson/document";
+import { BlockAnnotation } from "@atjson/document";
 
-export class HorizontalRule extends ObjectAnnotation {
+export class HorizontalRule extends BlockAnnotation {
```

Comment:
👍 matches the html element `<hr>` which is considered flow content like `<p>` and not considered phrasing content